sdk/@launchdarkly/observability-python/pyproject.toml

```diff
@@ -98,12 +100,6 @@ packages = [{ include = "ldobserve" }]
 requires = ["poetry-core>=2.0.0,<3.0.0"]
 build-backend = "poetry.core.masonry.api"
 
-[project.entry-points."opentelemetry_configurator"]
```

Comment:
Removed the distro. We will not want customers to install a distro if we want auto-instrumentation to work.